# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../glew/x11.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- glew/config/Makefile.linux_orig	2005-11-16 06:44:35.000000000 +0100
+++ glew/config/Makefile.linux	2005-11-16 06:44:50.000000000 +0100
@@ -6,7 +6,7 @@
 endif
 PICFLAG = -fPIC
 LDFLAGS.SO = -shared -soname $(LIB.SONAME)
-LDFLAGS.EXTRA = -L/usr/X11R6/lib
+LDFLAGS.EXTRA = -L/usr/X11/lib
 LDFLAGS.GL = -lXmu -lXi -lGLU -lGL -lXext -lX11
 LDFLAGS.STATIC = -Wl,-Bstatic
 LDFLAGS.DYNAMIC = -Wl,-Bdynamic
